app/assets/javascripts/oxalis/view/action-bar/save_button.js

```diff
@@ -29,29 +37,55 @@ class SaveButton extends React.PureComponent<Props, State> {
 
   savedPollingInterval: number = 0;
   _forceUpdate = () => {
```

Comment:
To get rid of the hourglass icon in the save button that is briefly displayed after saving, although the state was just successfully saved, we could call _forceUpdate after isBusy changed or was set to false (check for that in componentDidUpdate or so). Fine for me anyways, just wanted to mention it :)

Reply:
Good point! However, I'd like to postpone this until we get rid of the entire forceUpdate stuff (hopefully soon). It's quite tedious and maybe the flickering hourglass will serve as a reminder for us to improve this :)